manage.py

```diff
@@ -1,29 +1,107 @@
+import os
 import sys
+import click
 import logging
 
-from listenbrainz_spark.stats import user
-from listenbrainz_spark.recommendations import create_dataframes
-from listenbrainz_spark.recommendations import train_models
-from listenbrainz_spark.recommendations import recommend
-from listenbrainz_spark.recommendations import candidate_sets
+import listenbrainz_spark
+from listenbrainz_spark import path
+from listenbrainz_spark import utils
+from listenbrainz_spark import config
+from listenbrainz_spark import hdfs_connection
 
-if __name__ == '__main__':
-    if len(sys.argv) != 2:
-        print("Usage: manage.py <module_name>")
+from hdfs.util import HdfsError
+from py4j.protocol import Py4JJavaError
+
+@click.group()
+def cli():
+    pass
+
+@cli.command(name='init_dir')
+@click.option('--rm', is_flag=True, help='Delete existing directories from HDFS.')
+@click.option('--f', is_flag=True, help='Delete existing directories from HDFS recursively.')
```

Comment:
-f suggests "force". I think --recursive might be a better option here. Harder to get wrong too, which is more safe for us.